hologram_dashboard/fetcher/hologram_data_source_availability_fetcher.h

```diff
@@ -48,11 +49,23 @@ class HologramDataSourceAvailabilityFetcher: public HologramDataFetcher {
     // Poulates hologram_config_ and ends the program if the path provided leads
     // to wrong file or malformed file.
     void AcquireConfig(const std::string& config_file_path);
-    
+
+    FRIEND_TEST(FetcherTest, UpdateHistoryNewProto);
+    FRIEND_TEST(FetcherTest, UpdateHistorySameDay);
+    FRIEND_TEST(FetcherTest, UpdateHistoryIncrementHistory);
+    FRIEND_TEST(FetcherTest, UpdateHistoryOverflowHistory);
+    // Updates the history of a specified proto
+    void UpdateHistory (HologramDataAvailability& hda, std::time_t time, 
+        StatusType status);
+
+    FRIEND_TEST(FetcherTest, UpdateProto);
+    // Updates the proto for a data source given all the necessary information.
+    void UpdateProto(std::string system, std::time_t time, 
```

Comment:
The current .h file doesn't give me a clear picture about how you gonna organize the functions. 

Can you add a PR to check in the skeleton of this class first?

Reply:
Didn't you say not to check in private function skeletons if they're not implemented yet?

Reply:
As chatted offline, you need to show the workflow of your code to help readers understand.